other-identifiers/src/main/java/org/reactome/release/otherIdentifiers/PostStepChecks.java

```diff
@@ -0,0 +1,68 @@
+package org.reactome.release.otherIdentifiers;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.gk.model.GKInstance;
+import org.gk.model.ReactomeJavaConstants;
+import org.gk.persistence.MySQLAdaptor;
+
+import java.util.Collection;
+import java.util.concurrent.atomic.AtomicInteger;
+
+/**
+ *  This is a QA step that will be run after finishing an OtherIdentifiers run. For each species,
+ *  the count of ReferenceGeneProduct instances that have their 'otherIdentifier' attribute filled is found
+ *  between the current and previous database. If the previous database has a higher count, a warning message
+ *  is output and should be looked into. If the count is still relatively close,  it can probably be ignored.
+ */
+
+public class PostStepChecks {
+    private static final Logger logger = LogManager.getLogger();
+
+    public static void compareOtherIdentifierCounts(MySQLAdaptor dba, MySQLAdaptor dbaPrev) throws Exception {
+
+        Collection<GKInstance> speciesInstances = dba.fetchInstancesByClass(ReactomeJavaConstants.Species);
+
+        for (GKInstance speciesInstance : speciesInstances) {
+            int currentOtherIdentifierCount = getRGPInstanceOtherIdentifierCount(dba, speciesInstance);
+            int previousOtherIdentifierCount = getRGPInstanceOtherIdentifierCount(dbaPrev, speciesInstance);
+            if (currentOtherIdentifierCount < previousOtherIdentifierCount) {
+                logger.warn(speciesInstance.getDisplayName() + " has fewer ReferenceGeneProduct instances with OtherIdentifiers compared to previous release:  " +
+                        "release_current - " + currentOtherIdentifierCount + " -- release_previous - " + previousOtherIdentifierCount);
+            }
+        }
+    }
+
+    /**
+     * Wrapper method for determining other identifier count.
+     * @param dba MySQLAdaptor
+     * @param speciesInstance GKInstance pertaining to a specific species
+     * @return The other identifier count, as an int.
+     * @throws Exception
```

Comment:
If possible, it would be good to include what causes an exception to be thrown.  Usually, I just temporarily remove the `throws Exception` part of the method signature and see what part of the code throws the exception and document that a failure of that logic to complete successfully is what causes it.